doc/reference/iot.rst

```diff
@@ -0,0 +1,203 @@
+
+==================
+Internet of Things
+==================
+
+IoT Drivers allow any Odoo module to communicate in real-time with any device
+connected to the IoT Box. Communication with the IoT Box goes both ways, so the
+Odoo client can send commands to and receive information from any of the
+supported devices.
+
+To add support for a device, all we need is:
+
+- an `Interface`, to detect connected devices
```

Comment:
specify that an 'interface' detects all devices of a specific type
and that a driver communicate with a specific device